src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task{
+
+    protected String by;
```

Comment:
I think it would be better to keep your class variables as private. Do you have a reason to declare them as protected?